scipy/stats/qmc.py

```diff
@@ -0,0 +1,1368 @@
+# -*- coding: utf-8 -*-
+"""Quasi-Monte Carlo methods.
+
+Define function to generate sample of points in the unit hypercube.
+
+"""
+from abc import ABC, abstractmethod
+import copy
```

Comment:
please remove `copy`. where you use `copy.deepcopy` use can just use the ndarray `copy` method